conversions/infix_to_postfix.c

```diff
@@ -0,0 +1,192 @@
+/**
+ * Program documentation
+ * @name infix_to_postfix
+ * @brief Infix to Postfix Expression Conversion
+ * @details Convert Infixed expressions to Postfix expression.
+ * @author [Harsh Karande](https://github.com/harshcut)
+ */
+
+#include <stdio.h>
+
+// globally declared structure
+struct Stack
+{
+    char arr[10];  // static array of integers
+    int tos;       // stores index on topmost element in stack
+};
+
+// function headers
+void push(struct Stack *p, char ch);  // pust element in stack
+char pop(struct Stack *p);            // pop topmost element from the stack
+int isOprnd(char ch);                 // chack if element is operand or not
+int isEmpty(struct Stack s);          // check if stack is empty
+int prcnd(char op1, char op2);        // check operator precedence
+void convert(char infix[],
+             char postfix[]);  // convert infix to postfix expression
+
+/**
+ * @brief Main function
+ * @returns 0 on exit
+ */
+int main()
+{
+    char infix[20], postfix[20];
+
+    printf("Enter infix expression: ");
+    scanf("%s", infix);
+
+    convert(infix, postfix);
+    printf("Postfix expression is %s", postfix);
+
+    return 0;
+}
+
+/**
+ * @brief push function
+ * @returns void
+ */
+void push(struct Stack *p, char x)
+{
+    if (p->tos == 9)
+    {
+        printf("Stack Overflow!");
+        return;
+    }
+
+    p->tos += 1;
+    p->arr[p->tos] = x;
+}
+
+/**
+ * @brief pop function
+ * @returns x or \0 on exit
+ */
+char pop(struct Stack *p)
+{
+    char x;
+
+    if (p->tos == -1)
+    {
+        printf("Stack Underflow!");
+        return '\0';
+    }
+
+    x = p->arr[p->tos];
+    p->tos -= 1;
+
+    return x;
+}
+
+/**
+ * @brief isOprnd function
+ * @returns 1 or 0 on exit
+ */
+int isOprnd(char ch)
+{
+    if ((ch >= 65 && ch <= 90) || (ch >= 97 && ch <= 122) ||
+        (ch >= 48 && ch <= 57))
+    {
+        return 1;
+    }
+    else
+    {
+        return 0;
+    }
+}
+
+/**
+ * @brief isEmpty function
+ * @returns 1 or 0 on exit
```

Comment:
Undocumented parameter.

Reply:
Added parameter documentation for push function on line 103.